crates/runtime/src/embeddings/index/snapshots/runtime__embeddings__index__tests__scan_table_join_for_projection.snap

```diff
@@ -9,8 +9,8 @@ snapshot_kind: text
 | logical_plan  | Sort: my_vectored_table.pk DESC NULLS FIRST, fetch=5                                                                             |
 |               |   TableScan: my_vectored_table projection=[pk, another_column, body_embedding]                                                   |
 | physical_plan | SortPreservingMergeExec: [pk@0 DESC], fetch=5                                                                                    |
-|               |   CoalesceBatchesExec: target_batch_size=8192, fetch=5                                                                           |
-|               |     SortExec: TopK(fetch=5), expr=[pk@0 DESC], preserve_partitioning=[true]                                                      |
+|               |   SortExec: TopK(fetch=5), expr=[pk@0 DESC], preserve_partitioning=[true]                                                        |
+|               |     CoalesceBatchesExec: target_batch_size=8192                                                                                  |
 |               |       CoalesceBatchesExec: target_batch_size=8192                                                                                |
```

Comment:
A bit strange that we have two CoalesceBatchesExec in a row with the same target batch size? I don't think it should block this PR merging, but we should log a bug to investigate this.

Reply:
Issue: https://github.com/spiceai/spiceai/issues/7147